Samples/iOS-Swift/iOS-Swift.xcodeproj/project.pbxproj

```diff
@@ -1430,6 +1432,7 @@
 				PRODUCT_BUNDLE_IDENTIFIER = "io.sentry.sample.iOS-Swift";
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				PROVISIONING_PROFILE_SPECIFIER = "match Development io.sentry.sample.iOS-Swift";
+				"PROVISIONING_PROFILE_SPECIFIER[sdk=iphoneos*]" = "match AppStore io.sentry.sample.iOS-Swift";
```

Comment:
Are these extra values needed with the new cert and provisioning profile?

Reply:
It is the way Xcode 16 handled it. Better than manually changing the .xcodeproj file